.clang-format

```
@@ -0,0 +1,111 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Mozilla
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
```

```suggestion
AlignAfterOpenBracket: DontAlign
```
I think this looks better when you have a function name that is really long or has many parameters. For example, `DontAlign` gives,
```C++
void score_general_ce(Particle* p, int i_tally, int start_index,
  int filter_index, int i_nuclide, double atom_density, double flux)

FilterBinIter::FilterBinIter(const Tally& tally, bool end,
  std::vector<FilterMatch>* particle_filter_matches)
  : tally_ {tally}, filter_matches_ {*particle_filter_matches}
```
Instead of
```C++
void score_general_ce(Particle* p, int i_tally, int start_index,
                      int filter_index, int i_nuclide, double atom_density,
                      double flux)

FilterBinIter::FilterBinIter(const Tally& tally, bool end,
                             std::vector<FilterMatch>* particle_filter_matches)
  : tally_ {tally}, filter_matches_ {*particle_filter_matches}
```